serde/src/ser/mod.rs

```diff
@@ -1805,9 +1804,9 @@ pub trait SerializeMap {
     /// care about performance or are not able to optimize `serialize_entry` any
     /// better than this.
     ///
-    /// [`Serialize`]: ../trait.Serialize.html
-    /// [`serialize_key`]: #tymethod.serialize_key
-    /// [`serialize_value`]: #tymethod.serialize_value
+    /// [`Serialize`]: crate::Serialize
```

Comment:
Some lines can be removed altogether because corresponding items already in scope. For example, this line